rules/windows/lateral_movement_execution_from_tsclient_mup.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/11"
+ecs_version = ["1.6.0"]
+maturity = "development"
+query_schema_validation = false
+updated_date = "2020/11/11"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution from the RDP Shared MountPoint TSClient on the target host. This may indicate lateral movement
+attempt.
```

Comment:
```suggestion
Identifies execution from the Remote Desktop Protocol (RDP) shared mountpoint tsclient on the target host. This may indicate lateral movement
attempt.
```